doc/meson.build

```diff
@@ -1,4 +1,8 @@
- headerdir = join_paths(meson.source_root(), 'va')
+if meson.version().version_compare('>= 0.56')
+  headerdir = join_paths(meson.project_source_root(), 'va')
+else
+  headerdir = join_paths(meson.source_root(), 'va')
```

Comment:
Briefly looking I don't see neither of these functions in meson documentation. And can't check deprecation status. Maybe I just don't know where to look.